drupal8/docker-compose.yml

```diff
@@ -42,9 +44,13 @@ services:
     volumes:
       # Main code volume mount.
       - .:/var/www
+      # If you symlink a parent directory of files, be sure to set the destination
```

Comment:
This is a little awkwardly worded. Perhaps:

If a parent (ancestor?) of the files directory is a symlink, be sure to set the destination ...